NEWS.md

```diff
@@ -1,3 +1,6 @@
+# deltaccd 0.2.4
```

Comment:
fix version number and rebuild pkgdown site.

Reply:
Fixed!